style.css

```diff
@@ -1,6 +1,1178 @@
 
 @import url('https://fonts.googleapis.com/css?family=Poppins');
 
+@media screen and (min-width: 769px) {
+  .ul {
```

Comment:
- [ ] Well done on respecting the requirement, but I think your media query starts from `769px`and the output on the browser is not responsive at all, please take a look at this record and make the necessary changes to improve your styling

![5](https://user-images.githubusercontent.com/77843050/213782072-72164fc7-5e8f-4711-934d-032426498cbe.gif)

- [ ] Please make sure your hero text uses the same font weight property as per the original design

- [ ] Your social media icons are not well styled, please compare the 2 images and make the necessary changes

<img width="948" alt="1" src="https://user-images.githubusercontent.com/77843050/213782709-9ed451de-4776-4e23-816f-f9517884b18e.png">